### DISCUSSION TITLE
Welcome to ImprovedBlizzardUI Discussions!

### DISCUSSION THREAD

**Discussion by kaytotes** [kaytotes/ImprovedBlizzardUI#166] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Backupoffmy** [comment]
Any new planned updates for the Shadowlands release version? Been following your addon since its inception and i love what you do with it! It's amazing brother! Do you have a Patreon by chance?

**Reply by kaytotes** [reply]
Right now I have a couple things on the bucket list so to speak. 

1. The focus frame is spewing errors for me lately but trying to figure out if that's this addon or taint from another.
2. I still really really hate the action bars blizzard has and want to try and get us back to where I had it Pre-8.0
3. As above with the Party frames which Blizzard broke in 8.2.5

As for new functionality 

1. Ability to see the Great Vault and check your progress from anywhere.
2. Hopefully looking at some of the extra unit frames such as Boss Frames and Arena Frames.

I do not have a patreon.

**Comment by Backupoffmy** [comment]
I appreciate the reply!! Keep it up! Been using your UI addon since
inception!?
